### ISSUE TITLE
[BUG]: Failed to load SANA model

### ISSUE THREAD

**Issue by ps8-dev** [NVIDIA-AI-Blueprints/3d-object-generation#26]
### Version

58d7331

### Describe the bug.

Models get loaded but SANA (Gemma2) model does not load. No images visible in web interface

### Minimum reproducible example

```shell

```

### Relevant log output

```shell
2025-10-05 12:06:19,904 - services.image_generation_service - INFO - Loading SANA model...
check_services_health
global _in_workspace_mode True
current_counter 1
Loading pipeline components...:   0%|                                                            | 0/5 [00:00<?, ?it/s]`torch_dtype` is deprecated! Use `dtype` instead!
Loading pipeline components...:  20%|██████████▍                                         | 1/5 [00:00<00:00, 68.91it/s]
2025-10-05 12:06:20,312 - services.image_generation_service - ERROR - Error loading SANA model: Gemma2Model.__init__() got an unexpected keyword argument 'offload_state_dict'
Timestamp after generate_images_for_objects: 1759658780.3134086
2025-10-05 12:06:20,313 - root - INFO - Total GPU VRAM: 31.84 GB
2025-10-05 12:06:20,313 - root - INFO - VRAM threshold: 16 GB
2025-10-05 12:06:20,313 - root - INFO - VRAM check result: False
Image generation failed: Failed to load SANA model
```

### Full env printout

```shell

```

### Code of Conduct

- [x] I agree to follow __THIS PROJECT__'s Code of Conduct
- [x] I have searched the [open bugs](https://github.com/jarmak-nv/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

**Comment by jpenningCA** [comment]
Hi @ps8-dev , thanks for letting us know. We're working on a new release which we hope will address this bug.

**Comment by jpenningCA** [comment]
@ps8-dev  can you to test with latest release?

**Comment by ps8-dev** [comment]
I'm getting this error

Starting installation process... of 231,0 KB (0,0 MB/s)                                                              ]
Setting up trellis submodule...
The system cannot find the path specified.
fatal: not a git repository (or any of the parent directories): .git
Using conda environment name: trellis
Configuration:
  Conda environment: trellis
  USE_NATIVE_LLM: True
  USE_NATIVE_TRELLIS: True
Checking for Conda...
Conda command returned errorlevel: 0
Conda check passed, proceeding to file checks...
[ERROR] requirements-torch.txt not found
Press any key to continue . . .
